syft/frameworks/torch/pointers/pointer_tensor.py

```diff
@@ -239,6 +349,9 @@ def item(self) -> None:
             "so you can be safely getting the item you need."
         )
 
+    def __eq__(self, other):
```

Comment:
I don't know exactly, but I think comparison operator are a bit tricky to handle and moving from __ eq__ to .eq really helped
That's what we do in native.py and because we now operate on pointers (when wrappers are removed), we need them to have the same trick